domain/src/test/java/org/oppia/android/domain/oppialogger/analytics/PerformanceMetricsLoggerTest.kt

```diff
@@ -236,6 +243,21 @@ class PerformanceMetricsLoggerTest {
     assertThat(loggedEvent.isAppInForeground).isEqualTo(isAppInForeground)
   }
 
+  @Test
+  fun testLogger_inDefaultState_logMultipleEvents_doesNotLogEvent() {
```

Comment:
This test name is a bit confusing--it seems to imply that events are logged, but also not. Should the last part be "doesNotLogAnyPerformanceMetric"?

Reply:
changed test name to `testLogger_inDefaultState_tryToLogMultipleEvents_doesNotLogAnyPerformanceMetric`.